src/identity/SolidOidcIdentityProvider.ts

```diff
@@ -0,0 +1,87 @@
+import { Provider } from 'oidc-provider';
+
+/**
+ * An Extension of node-oidc-provider with configurations specifically for
+ * Solid OIDC
+ */
+export default class SolidOidcIdentityProvider extends Provider {
+  public constructor() {
+    // TODO (@jackson) [>1.0.0]: finish this.
```

Comment:
Shall we indicate explicitly what is finished and what isn't?

Reply:
Sorry, most of these comments are notes to self. They will be removed for the merge into master. Also notice that all this code is VERY messy. I'm going to be cleaning that up now that I got login working.